Tasks/PublishBuildArtifacts/task.json

```diff
@@ -50,9 +50,9 @@
       "name": "TargetPath",
       "type": "string",
       "label": "File share path",
-      "defaultValue": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)",
```

Comment:
Did we remove this because it wasn't clear to users they needed to update it to have a successful build?

Reply:
@hashtagchris Yes, I was probably thinking that examples shouldn't be used as invalid default values of tasks.  If the value must be changed, the user has no indication of it until the build fails.  In this case, the example was moved to the tooltip.